setuptools/tests/test_find_packages.py

```diff
@@ -7,10 +7,15 @@
 
 import pytest
 
-import setuptools
 from setuptools import find_packages
 
-find_420_packages = setuptools.PEP420PackageFinder.find
+try:
+  from setuptools import find_packages_ns
+except:
+  # just catch
+  pass
+
+py33_upwards = pytest.mark.xfail(sys.version_info < (3,3), reason="Test runs on Python >= 3.3 only")
```

Comment:
This should be `skipif` not `xfail`.

`xfail` indicates that it is currently a failing test that needs to be fixed. `skipif` indicates that the tested behavior isn't intended to occur under the `skipif` conditions.

Reply:
@pganssle i found that all other conditional tests use xfail, as in expected failure. but i can make this a skipif.

Reply:
done

Reply:
Hm, I was planning on going through and checking on those other `xfail` tests because I did notice a few of them passing inappropriately.

I think it's a bit strange to use `xfail` instead of `skipif`, because `xfail` is more useful if you would like it to start passing at some point in the future and it's unnecessary to run tests where you don't actually *care* whether or not they pass.